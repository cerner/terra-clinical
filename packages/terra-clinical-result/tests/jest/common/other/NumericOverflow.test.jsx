import React from 'react';
/* eslint-disable-next-line import/no-extraneous-dependencies */
import { shallowWithIntl } from 'terra-enzyme-intl';
import NumericOverflow from '../../../../src/common/other/_NumericOverflow';

describe('NumericOverflow', () => {
  it('should render a default NumericOverflow', () => {
    const result = shallowWithIntl(<NumericOverflow />).dive();
    expect(result).toMatchSnapshot();
  });
});
