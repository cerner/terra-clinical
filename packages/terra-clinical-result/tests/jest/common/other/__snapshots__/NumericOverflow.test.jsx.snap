// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumericOverflow should render a default NumericOverflow 1`] = `
<span
  className="numeric-overflow"
>
  Terra.clinicalResult.viewResults
</span>
`;
